package.json

```diff
@@ -43,7 +43,7 @@
     "stylelint": "^9.0.0"
   },
   "peerDependencies": {
-    "stylelint": "^8.4.0"
+    "stylelint": ">=8.4.0 || <10.0.0"
```

Comment:
Argh, I think I've been caught out by this myself previously :(

Do you think `>=8.4.0` would work without the `|| <10.0.0`?